src/services/expenses.services.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { Op } = require('sequelize');
+
+const { sequelize } = require('./database.service');
+
+const Expense = sequelize.define('Expense', {
+  userId: {
+    type: DataTypes.INTEGER,
+    field: 'user_id',
+    allowNull: false,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    field: 'spent_at',
+    allowNull: false,
+    defaultValue: DataTypes.NOW,
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.STRING,
+  },
+}, {
+  tableName: 'expenses',
+});
+
+const normalizeExpense
+  = ({ id, userId, spentAt, title, amount, category, note }) => {
+    return {
+      id, userId, spentAt, title, amount, category, note,
+    };
+  };
```

Comment:
what's the purpose of this function? there is no business logic inside

Reply:
this function need to excluding unnecessary data from database